### PR TITLE
Added Caution Module

### PR DESCRIPTION
**Pull request by digitalripperynr**
This PR adds the Caution Module.

It includes : 

- A small ship (cautionSmall)
- A medium ship (cautionMedium)
- A large ship (cautionBig)
- A capital ship (cautionCapital)
- A station (cautionStation)
- 2 Machine guns (cautionMachine and heartAttack)
- A projectile (Red Blood Cells)

### CODE REVIEW

**Comment by GooeyHub** [comment]
Can one of the admins please verify this patch?

File: modules/caution/assets/ships/cautionMedium/cautionMedium.json [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+    "type": "std",
+    "price": 900,
+    "ability": {
+        "type": "teleport",
+        "angle": 180,
+        "rechargeTime": 1
+    },
+    "e1Pos": "0.02 0.31",
+    "e2Pos": "0.02 0.69",
+    "displayName": "Caution11",
```
Comment (vampcat) [inline modules/caution/assets/ships/cautionMedium/cautionMedium.json]: Caution11 ?

File: modules/caution/assets/ships/cautionMedium/cautionMedium.json [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+        "origin": {
+            "x":0.5,
+            "y":0.5
+        },
+            "polygons":[],
+            "circles":[],
+            "shapes": [
+                {
+                  "type":"POLYGON",
+                  "vertices": [
```
Comment (vampcat) [inline modules/caution/assets/ships/cautionMedium/cautionMedium.json]: Collision mesh is wrong, as pointed out in irc.

**Comment by vampcat** [comment]
You haven't even enabled the module in game. How do you even expect me to test this?

**Comment by digitalripperynr** [comment]
Hey! I just fixed the collision mesh!
Please review

**Comment by digitalripperynr** [comment]
It got messed up again
Please wait before I clean it up

**Comment by digitalripperynr** [comment]
Done!
Please review...

**Comment by digitalripperynr** [comment]
Fixed the issues listed by @vampcat
Please review.

**Comment by digitalripperynr** [comment]
The `cautionMachine` is a modified blaster that shoots plasma cells at blazing fast speeds with low reload times.
_This is for GCI DestSol Series Part 2_

**Comment by digitalripperynr** [comment]
The cautionBig is a large ship in the module. The idea is to make this overkill by maximizing both attack and hit points.

This is for GCI DestSol Series Part 2

**Comment by vampcat** [comment]
@GooeyHub: ok to test

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by digitalripperynr** [comment]
The `cautionCapital` is the capital ship for the module. Code Name: No Mercy

_The ship firing plasma cells_
screenshot 203

_The collision mesh_
screenshot 202

_This is for GCI DestSol Series Part 3_

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by digitalripperynr** [comment]
The `cautionSmall` is the small ship for the module. Code Name: Fighter

_The ship firing plasma cells_
screenshot 206


_The collision mesh_
screenshot 204


_This is for GCI DestSol Series Part 4_

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by digitalripperynr** [comment]
The `cautionStation` is the station for the module.

_The collision mesh_



_This is for GCI DestSol Series Part 4_

**Comment by GooeyHub** [comment]
Can one of the admins please verify this patch?

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by digitalripperynr** [comment]
I accidentally closed the PR :P
Please ignore.

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by digitalripperynr** [comment]
The module is now finalized and ready to merge :)

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Pull request merged**